README.md

```diff
@@ -17,7 +17,7 @@ Add `:plug_caisson` to dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:plug_caisson, "~> 0.1.1"},
+    {:plug_caisson, "~> 0.2.0"},
```

Comment:
```suggestion
    {:plug_caisson, "~> 0.2"},
```

Reply:
That is not what we should have there, as it is against SemVer.

Reply:
Not quite sure what you mean as it would only match 0.2.x
https://hexdocs.pm/elixir/1.14/Version.html#module-requirements